### PR TITLE
Link accross the specification documents

### PR DESCRIPTION
**Pull request by no-reply**

### CODE REVIEW

File: otm-bridge.html [hunk 1]
```diff
@@ -191,7 +210,7 @@ <h3><dfn>Deposit Content</dfn></h3>
         <p>Requests that a set of content be deposited into the DDP</p>
         <ul>
           <li>Request Type: <code>POST</code></li>
-          <li>Request Path: <code>/deposit ? deposit-format=</code></li>
+          <li>Request Path: <code>/deposit ? deposit-format</code></li>
```
Comment (bbranan) [inline otm-bridge.html]: @no-reply I doubt this change (and the similar ones below it) was intentional. In any case, they will need to be reverted before merge.

File: audit-appendix.html [hunk 2]
```diff
@@ -23,7 +23,26 @@
     wg: "One to Many Working Group",
     wgURI: "https://wiki.duraspace.org/display/OTM",
     edDraftURI: "https://github.com/ucsdlib/otm-specs/blob/master/audit-appendix.html",
-    maxTocLevel: 3
+    maxTocLevel: 3,
+    otherLinks: [{
+      key: "OTM Specifications",
+      data: [{
+        value: "One to Many (OTM) Specifications Overview",
+        href: "https://ucsdlib.github.io/otm-specs/"
```
Comment (bbranan) [inline audit-appendix.html]: Maybe try "." as the link for the top level README.

File: otm-bridge.html (hunk 1 above)
Comment (no-reply) [inline otm-bridge.html]:
```suggestion
          <li>Request Path: <code>/deposit ? deposit-format=</code></li>

```

File: otm-bridge.html [hunk 3]
```diff
@@ -274,7 +293,7 @@ <h3><dfn>List Deposits</dfn></h3>
         calls <a>Complete Deposit</a>)</p>
         <ul>
           <li>Request Type: <code>GET</code></li>
-          <li>Request Path: <code>/deposit ? status=</code></li>
+          <li>Request Path: <code>/deposit ? status</code></li>
```
Comment (no-reply) [inline otm-bridge.html]:
```suggestion
          <li>Request Path: <code>/deposit ? status=</code></li>

```

File: otm-bridge.html [hunk 4]
```diff
@@ -509,7 +528,7 @@ <h3><dfn>List Restores</dfn></h3>
         <p>Retrieves a listing of all in-process restores</p>
         <ul>
           <li>Request Type: <code>GET</code></li>
-          <li>Request Path: <code>/restore ? status=</code></li>
+          <li>Request Path: <code>/restore ? status</code></li>
```
Comment (no-reply) [inline otm-bridge.html]:
```suggestion
          <li>Request Path: <code>/restore ? status=</code></li>
```

File: otm-bridge.html [hunk 5]
```diff
@@ -748,7 +767,7 @@ <h3><dfn>List Deletes</dfn></h3>
         <p>Retrieves a listing of all in-process deletes</p>
         <ul>
           <li>Request Type: <code>GET</code></li>
-          <li>Request Path: <code>/delete ? status=</code></li>
+          <li>Request Path: <code>/delete ? status</code></li>
```
Comment (no-reply) [inline otm-bridge.html]:
```suggestion
          <li>Request Path: <code>/delete ? status=</code></li>
```

Comment (bbranan) [reply otm-bridge.html]: Looks like the restored equals is missing here

**Comment by no-reply** [comment]
https://github.com/ucsdlib/otm-specs/issues/21

File: audit-appendix.html (hunk 2 above)
Comment (no-reply) [inline audit-appendix.html]:
```suggestion
        href: "."
```

File: otm-bridge.html [hunk 6]
```diff
@@ -29,7 +29,26 @@
             title: "The 'Basic' HTTP Authentication Scheme",
             href: "https://tools.ietf.org/html/rfc7617"
           }
-        }
+        },
+        otherLinks: [{
+          key: "OTM Specifications",
+          data: [{
+            value: "One to Many (OTM) Specifications Overview",
+            href: "https://ucsdlib.github.io/otm-specs/"
```
Comment (no-reply) [inline otm-bridge.html]:
```suggestion
            href: "."

```

File: otm-gateway.html [hunk 7]
```diff
@@ -29,7 +29,26 @@
             title: "The 'Basic' HTTP Authentication Scheme",
             href: "https://tools.ietf.org/html/rfc7617"
           }
-        }
+        },
+        otherLinks: [{
+          key: "OTM Specifications",
+          data: [{
+            value: "One to Many (OTM) Specifications Overview",
+            href: "https://ucsdlib.github.io/otm-specs/"
```
Comment (no-reply) [inline otm-gateway.html]:
```suggestion
            href: "."
```

File: preservation-workflow.html [hunk 8]
```diff
@@ -33,7 +33,26 @@
         title: "Chronopolis OCFL Evaluation"   ,
         href: "https://wiki.duraspace.org/display/OTM/Chronopolis+OCFL+Evaluation"
       }
-    }
+    },
+    otherLinks: [{
+      key: "OTM Specifications",
+      data: [{
+        value: "One to Many (OTM) Specifications Overview",
+        href: "https://ucsdlib.github.io/otm-specs/"
```
Comment (no-reply) [inline preservation-workflow.html]:
```suggestion
        href: "."
```

File: otm-bridge.html [hunk 9]
```diff
@@ -192,6 +212,7 @@ <h3><dfn>Deposit Content</dfn></h3>
         <ul>
           <li>Request Type: <code>POST</code></li>
           <li>Request Path: <code>/deposit ? deposit-format=</code></li>
+
```
Comment (no-reply) [inline otm-bridge.html]:
```suggestion
```

File: otm-bridge.html [hunk 10]
```diff
@@ -275,6 +296,7 @@ <h3><dfn>List Deposits</dfn></h3>
         <ul>
           <li>Request Type: <code>GET</code></li>
           <li>Request Path: <code>/deposit ? status=</code></li>
+
```
Comment (no-reply) [inline otm-bridge.html]:
```suggestion
```

**Pull request merged**